src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: Create and manage your accounts
```

Comment:
I think we want something that it makes it more clear about what the goal is for this doc --> "Account setup best practices"

Reply:
Good idea!